### PR TITLE
opf-logo-and-draft-status

### PR DESCRIPTION
**Pull request by Asbjoedt**

### CODE REVIEW

File: Draft v1.0/Specification.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
+<img src="https://openpreservation.org/wp-content/uploads/2019/11/opf-site-logo.png" width="300" alt="Open Preservation Foundation logo">
```
Comment (carlwilson) [inline Draft v1.0/Specification.md:1]: Use the markdown syntax for this please and chose an image of the right size, although you can add classes to GH markdown if memory serves.

**Review by carlwilson — CHANGES_REQUESTED** [review]

File: Draft v1.0/Specification.md (hunk 1 above)
Comment (Asbjoedt) [reply Draft v1.0/Specification.md:1]: If you don't want HTML to conrtrol the width, how do you propose we host an image file with the exact width?

**Review by carlwilson — COMMENTED** [review]
No logo?

**Comment by carlwilson** [comment]
There's conflict here ;)

**Comment by Asbjoedt** [comment]
New commit:
- changed logo from HTML to markdown (with help from Georgia)
- resolved the conflict

**Review by carlwilson — APPROVED** [review]

**Pull request merged**